spec/models/like_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'Validations For the Like model' do
+    before(:each) do
+      @user = User.create(name: 'Daniel', photo: 'image_link', bio: 'Developer from Nigeria')
+      @post = Post.create(author: @user, title: 'My title', text: 'My text')
+      @like = Like.create(author: @user, post_id: @post.id)
+    end
+
+    it 'if author_id is present' do
+      @like.author_id = false
+      expect(@like).not_to be_valid
+    end
+
+    it 'if post_id is present' do
+      @like.post_id = nil
+      expect(@like).to_not be_valid
+    end
+
+    it 'like must be present' do
+      expect(@post.likes.length).to eq 0
+    end
+  end
+end
```

Comment:
- [ ] You should add a test for the `update_likes_counter` method. Kindly add it. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec_n%2B1.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.)